src/lib/tls/tls_session.cpp

```diff
@@ -83,6 +85,7 @@ Session::Session(const byte ber[], size_t ber_len)
         .decode_integer_type(side_code)
         .decode_integer_type(fragment_size)
         .decode(m_extended_master_secret)
+        .decode(m_encrypt_then_mac)
```

Comment:
This changes the format of the session (which is fine at this point in the release cycle). But when doing so increase the version number encoded in the session.